### ISSUE TITLE
I found a bug using the demo!

### ISSUE THREAD

**Issue by bounceme** [shapesecurity/shift-fuzzer-js#11]
> Attempted to assign to readonly property.

Here's the script:

``` js
"¢";
"Bº[m";
"{";
'e"P';
"¸zÌ";
`j=${this}Å=8á${wslmsexi++}Ò~`;
let tyxsmyxk = 2e308;
for (var [] in new this(...3.671609666198493e83, ...[] = `{-`).b) while (null.jqvdbt **= 374561774239744e1) null;
for (tyvmyoe of ">Î®") for ([, {public, implements = function* () {
  "õ";
  "";
  "vì9";
  ">[";
}, i = function* arguments() {
  "";
  "ÇÞ";
  "ês";
  "";
  "µ¯";
}}] in 2e308([...package], e.private / /()+?/guy, let.enum instanceof "ø", 2e308 in (r >>= this))[null]) break;
debugger;
```

**Comment by michaelficarra** [comment]
Hmm, I can't reproduce. That parses fine for me as a Script.

**Comment by bakkot** [comment]
I expect this was the same issue as #13, which was resolved in shapesecurity/shift-parser-js#302 and shapesecurity/shift-website@c71f0fe.
